rasa/core/policies/ted_policy.py

```diff
@@ -541,7 +541,8 @@ def __init__(
         self.action_acc = tf.keras.metrics.Mean(name="acc")
         self.metrics_to_log += ["loss", "acc"]
 
-        self.all_labels_embed = None  # needed for efficient prediction
+        # needed for efficient prediction
+        self.all_labels_embed: Optional[Tuple[tf.Tensor, tf.Tensor]] = None
```

Comment:
judging based on [this line](https://github.com/RasaHQ/rasa/blob/077d5dfa6140771113cd06dffd068d2c3d36e446/rasa/nlu/classifiers/diet_classifier.py#L1814) I'd say it should be 
```suggestion
        self.all_labels_embed: Optional[tf.Tensor] = None
```